ngraph/core/include/ngraph/op/log_softmax.hpp

```diff
@@ -0,0 +1,62 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v5
+        {
+            class NGRAPH_API LogSoftmax : public Op
+            {
+            public:
+                NGRAPH_RTTI_DECLARATION;
+                LogSoftmax()
+                    : m_axis(0)
```

Comment:
You should call the `Op` class constructor here if you're planning to implement the default constructor yourself. I'd suggest to move the axis initialization to the line where the member is defined and just do `LogSoftmax() = default;` in here.